cockpit/build.gradle

```diff
@@ -4,11 +4,13 @@ apply plugin: 'kotlin-android-extensions'
 apply from: rootProject.file('gradle/publishing.gradle')
 
 buildscript {
+    ext.cockpit_core_version = '1.1.1-SNAPSHOT'
```

Comment:
What about extracting `cockpit_core_version` to `dependecies.gradle`?
`ext {
    ext.cockpit_core_version = '1.1.1-SNAPSHOT'
}`